Cypress/cypress/integration/Parallel_Tests/Create_Workflow/customWorkflow.spec.js

```diff
@@ -0,0 +1,108 @@
+/// <reference types="Cypress" />
+import * as workflows from "../../../fixtures/Workflows.json";
+import * as user from "../../../fixtures/Users.json";
+
+describe("Testing the workflow creation wizard using PreDefined Experiments", () => {
```

Comment:
This should not be called PreDefined since the inner tests are calling custom workflows